src/internal.d.ts

```diff
@@ -165,9 +165,8 @@ export interface Component<P = {}, S = {}>
 	constructor: ComponentType<P>;
 	state: S; // Override Component["state"] to not be readonly for internal use, specifically Hooks
 
+	_flags: number;
```

Comment:
CC @marvinhagemeister it looks like _prevState from devtools uses __u as well can we swap that?